src/cljs/org/broadinstitute/firecloud_ui/page/workspaces.cljs

```diff
@@ -66,11 +66,19 @@
       (:data props)])})
 
 
+(defn- filter-workspaces [f workspaces]
+  (case f
+    :all workspaces
+    :complete  (filter (fn [ws] (= "Complete"  (ws "status"))) workspaces)
```

Comment:
I always turn this alignment style off in my IDE. Do you think it adds readability?